preview/Win7Msix/Win7MSIXInstaller/MsixRequest.hpp

```diff
@@ -74,6 +79,8 @@ class MsixRequest
         return true;
     }

```

Comment:
make these also inline

Reply:
Added inline to methods executing minimal code.